m4_sessions/1711-inning/Group_1.md

```diff
@@ -1,9 +1,22 @@
 Group One
 
-Session Title: Description: Why should students come to this session?
+Session Title: Net Neutrality
+
+Description: Why should students come to this session?
+To Learn about Net Neutrality and how it affects us as web developers.  
+Learn about Internet control and censorship by internet service providers.
 
 Session Outcomes: What will students leave with or gain from this session?
+A greater understanding of this issue and how it will affect everyone.
+How to act and what to do fix it?
 
 Session Outline: Include the flow of the session and the time for each section.
 
-Names of group members:
+0 - 15 minutes: Everyone introduction
+15-25: video & article
+25-40: small group discussion
+40-50: large group discussion
+50-75: Talk about the future and actions we can do 
```

Comment:
I would recommend pushing everyone to write or to share their individual thoughts.  Also you each should be prepared to share in case there are not a lot of new ideas put forth.